src/src/pages/Dashboard/components/QuickStart/QuickStart.tsx

```diff
@@ -28,24 +31,22 @@ function QuickStart() {
           tint={100}
           className='QuickStart__section__title'
         >
-          Integrate Aim with your code
+          Integrate FasttrackML with your code
         </Text>
         <CodeBlock
-          code={`from aim import Run
+          code={`from mlflow import log_metric, log_param
 
-# Initialize a new run
-run = Run()
+# Set FasttrackML tracking server
+mlflow.set_tracking_uri("${fasttrack_server}")
```

Comment:
This won't run without importing the module or the function first

Reply:
For this, I have now included set_tracking_uri in the import 

```python
from mlflow import log_metric, log_param, set_tracking_uri

# Set FastTrackML tracking server
set_tracking_uri("${fasttrack_server}")
```